### DISCUSSION TITLE
Crash detection

### DISCUSSION THREAD

**Discussion by CorruptionVR** [copy/v86#956] [Q&A]
hey, is crash detection possible? because I think that it would be useful to know if and when the VM crashed

**Comment by spetterman66** [comment]
If you use v86's debug mode, and then open your web browser's developer tools console, you will see when an operating system crashes, and details.

**Reply by Alsefina** [reply]
debug mode?

**Reply by CorruptionVR** [reply]
Scroll to the bottom of the page, and you will see a link that says "enable debug"
